pkg/core/native/native_neo.go

```diff
@@ -32,6 +32,7 @@ type NEO struct {
 	gasPerBlock        atomic.Value
 	gasPerBlockChanged atomic.Value
 
+	initDone       bool
```

Comment:
Yet another flag? Can we arrange to live without it?

Reply:
We can reuse e.g. `gasPerBlockChanged` and check if it is `nil` in `OnPersist`, but it seems a bit clunky.

Reply:
Fixed.